__tests__/wallet.js

```diff
@@ -1,5 +1,5 @@
 const ecc = require('tiny-secp256k1')
-const { BIP32Factory } = require('bip32')
```

Comment:
lint ughhh

Reply:
shall we add .eslint as dev tool with prettier plugin?

Reply:
yes whatever you think is best for linting - i am not very well-versed in lint tools so i trust your judgment

Reply:
took a note